woocommerce/woocommerce-android
# [Better Tablet Products] Implement selection of a product

Closes: #10726 
Closes: #10779 


### Description

The PR adds a selection of product switches and product details fragment. On top of that I replaced implicit work mode selection of the product details framgnet via passing flags and product id 0 or not 0 to a explicite

### Testing instructions

* Make sure that with flag OFF both tablet and phone work in single pane mode
* Make sure that with flag ON tablet works in two pane mode (with bugs as this is WIP) and phone work in single pane mode

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/63980889-2930-45c4-8e61-7e10c8a00ca8
